README.md

```diff
@@ -842,6 +852,7 @@ ever.
 * [Making WebGL Dance](https://www.youtube.com/watch?v=GNO_CYUjMK8) [00:30:55] by **Steven Wittens** (2013)
 * [Making things with maths](https://www.youtube.com/watch?v=Zkx1aKv2z8o) [00:53:45] by **Steven Wittens** (2013)
 * [Simit: A Language for Computing on Sparse Systems when Performance Matters](https://www.youtube.com/watch?v=raPkxhHy5ro) [00:58:37] by **Fredrik Kjolstad** (2016)
+* [The Glitching Hour](https://www.youtube.com/watch?v=2HoMsbxY9zA) [00:22:16] by **Amy Wibowo** (2018)
```

Comment:
in Chrome (my main browser) it's 22:16 and I guess you're using Firefox because Firefox shows 22:17 for me. Maybe I should use that `add-youtube-durations` script for the real answer...

Reply:
No, it’s fine. I just thought it was fun and curious!

Reply:
it's actually 22:16 + 22 frames, weird to know that the length is calculated by browser JavaScript (hence the rounding difference) rather than stored on YouTube's servers. I guess I'll round this up?

Reply:
While I appreciate the scientific accuracy, I don’t feel strongly either way. I’m confident that whatever you do, you’ll make a great decision!

Reply:
your "it's fine" comment never loaded for some reason, but this one i'll leave as-is :)